src/surface/flight_control/flight_control/multiplexer.py

```diff
@@ -58,33 +58,36 @@ def __init__(self) -> None:
         )
 
         self.rc_pub = self.create_publisher(
-            OverrideRCIn, 'mavros/rc/override', QoSPresetProfiles.DEFAULT.value
+            ManualControl, 'mavros/manual_control/send', QoSPresetProfiles.DEFAULT.value
         )
 
     @staticmethod
     def apply(msg: PixhawkInstruction, function_to_apply: Callable[[float], float]) -> None:
         """Apply a function to each dimension of this PixhawkInstruction."""
         msg.forward = function_to_apply(msg.forward)
-        msg.vertical = function_to_apply(msg.vertical)
+        msg.vertical = msg.vertical
         msg.lateral = function_to_apply(msg.lateral)
         msg.pitch = function_to_apply(msg.pitch)
         msg.yaw = function_to_apply(msg.yaw)
         msg.roll = function_to_apply(msg.roll)
 
     @staticmethod
-    def to_override_rc_in(msg: PixhawkInstruction) -> OverrideRCIn:
+    def to_manual_control(msg: PixhawkInstruction) -> ManualControl:
         """Convert this PixhawkInstruction to an rc_msg with the appropriate channels array."""
-        rc_msg = OverrideRCIn()
+        rc_msg = ManualControl()
```

Comment:
Rename `rc_msg` (and anything else referencing rc stuff) to `manual_control_msg`, etc.